### PR TITLE
MODLD-351: Replace WorkReference API model with Work

### PR DESCRIPTION
**Pull request by askhat-abishev**

### CODE REVIEW

File: src/main/java/org/folio/linked/data/mapper/dto/monograph/work/WorkReferenceMapperUnit.java [hunk 1]
```diff
@@ -1,45 +1,55 @@
...
+import org.folio.linked.data.mapper.dto.monograph.common.NoteMapper;
 import org.folio.linked.data.model.entity.Resource;
 import org.folio.linked.data.repo.ResourceRepository;
 import org.springframework.stereotype.Component;
 
 @Component
 @RequiredArgsConstructor
-@MapperUnit(type = WORK, predicate = INSTANTIATES, dtoClass = WorkReference.class)
+@MapperUnit(type = WORK, predicate = INSTANTIATES, dtoClass = Work.class)
 public class WorkReferenceMapperUnit implements SingleResourceMapperUnit {
```
Comment (aleksei-pronichev) [inline src/main/java/org/folio/linked/data/mapper/dto/monograph/work/WorkReferenceMapperUnit.java:30]: Do we need to rename this class?

**Review by aleksei-pronichev — APPROVED** [review]

File: src/main/java/org/folio/linked/data/mapper/dto/monograph/work/WorkReferenceMapperUnit.java (hunk 1 above)
Comment (askhat-abishev) [reply src/main/java/org/folio/linked/data/mapper/dto/monograph/work/WorkReferenceMapperUnit.java:30]: yes, we do

Comment (PBobylev) [reply src/main/java/org/folio/linked/data/mapper/dto/monograph/work/WorkReferenceMapperUnit.java:30]: but Instance's field name is still the "_workReference", so we should keep it as is I think

Comment (askhat-abishev) [reply src/main/java/org/folio/linked/data/mapper/dto/monograph/work/WorkReferenceMapperUnit.java:30]: sorry, I've misunderstood the question initially

**Review by PBobylev — APPROVED** [review]

**Pull request merged**